### ISSUE TITLE
Ignore .DS_Store files

### ISSUE THREAD

**Issue by martinrybak** [kevmoo/preload#1]
The following `.DS_Store` files were included in the generated `index.html`:
```
<link rel="preload" href="assets/.DS_Store" as="fetch" crossorigin>
<link rel="preload" href="packages/foo/.DS_Store" as="fetch" crossorigin>
```

**Comment by kevmoo** [comment]
Yes!

**Comment by kevmoo** [comment]
Just released `1.1.0` with this fix @martinrybak
